command/ca/certificate.go

```diff
@@ -102,6 +116,30 @@ the complete set of subjective alternative names in the token 1:1. Use the '--sa
 flag multiple times to configure multiple SANs. The '--san' flag and the '--token'
 flag are mutually exlusive.`,
 			},
+			cli.BoolFlag{
+				Name:  "acme",
+				Usage: `Use the ACME protocol to get a certificate.`,
+			},
+			cli.BoolFlag{
+				Name: "standalone",
+				Usage: `Run a standalone webserver for ACME authentication. Must be used in conjunction
+with the --acme flag.`,
+			},
+			cli.StringFlag{
+				Name: "webroot",
+				Usage: `Run a standalone webserver for ACME authentication. Must be used in conjunction
+with the --acme flag.`,
+			},
+			cli.StringSliceFlag{
+				Name: "contact",
+				Usage: `Email addresses for contact as part of the ACME protocol. These contacts
+may be used to warn of certificate expration or other certificate lifetime events.`,
+			},
+			cli.IntFlag{
+				Name:  "http-port",
+				Usage: `Use a non-standard http port behind a reverse proxy or load balancer. (default is 80)`,
```

Comment:
I've added later a comment suggesting that this could be an address perhaps called `--listen`

Reply:
I'm not sure I like --listen because when we eventually allows tls support for challenges we'd have a https-port. We could call it `--listen` and `--https-listen`, but I'm not sure how much better that is.

Reply:
So use `--http-listen=<address>` and `--https-listen=<address>`

Reply:
I can do that. For some background, the reason I went with `--http-port` (and `--https-port`) is that's what acme.sh and lego cli use.

Reply:
Generay is allways better to be able to specify the interface you want to listen on, an address allows you to do that.